src/assets/wise5/authoringTool/project-authoring/project-authoring.component.html

```diff
@@ -56,52 +56,36 @@
   </button>
 </div>
 <div class="all-nodes-div">
-  <div *ngFor="let item of items">
+  <div *ngFor="let lesson of lessons">
     <project-authoring-lesson
-      *ngIf="isGroupNode(item.id)"
-      [item]="item"
-      [disableSelection]="stepNodeSelected"
-      (selectNodeEvent)="selectNode()"
-      [showPosition]="true"
-    ></project-authoring-lesson>
-    <project-authoring-step
-      *ngIf="!isGroupNode(item.id)"
-      [item]="item"
-      (selectNodeEvent)="selectNode()"
-      [disableSelection]="groupNodeSelected"
+      [lesson]="lesson"
+      [nodeTypeSelected]="nodeTypeSelected"
```

Comment:
ProjectAuthoringLesson and ProjectAuthoringStep shouldn't have to know about NodeTypeSelected- they should only need to know if they need to disable selection?
```suggestion
      [disableSelection]="nodeTypeSelected === 'step'"
```